builtin/logical/ssh/path_keys.go

```diff
@@ -0,0 +1,97 @@
+package ssh
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/vault/logical"
+	"github.com/hashicorp/vault/logical/framework"
+)
+
+func pathKeys(b *backend) *framework.Path {
+	return &framework.Path{
+		Pattern: "keys/(?P<name>\\w+)",
```

Comment:
Done!
@armon Should we warn the user if the chosen name is not matching the pattern accepted by the endpoint?